webapp/components/NotificationsTable/NotificationsTable.spec.js

```diff
@@ -3,7 +3,7 @@ import { config, mount, RouterLinkStub } from '@vue/test-utils'
 import Vuex from 'vuex'
 import NotificationsTable from './NotificationsTable'
 
-import { notifications } from '~/components/utils/Notifications'
+import { testNotifications } from '~/components/utils/Notifications'
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-23T19:27:46Z" title="Thursday, January 23rd 2020, 8:27:46 pm +01:00">Jan 23, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -3,7 +3,7 @@ import { config, mount, RouterLinkStub } from '@vue/test-utils'
 import Vuex from 'vuex'
 import NotificationsTable from './NotificationsTable'
 
-import { notifications } from '~/components/utils/Notifications'
+import { testNotifications } from '~/components/utils/Notifications'
```


don't understand this change...

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-02-05T11:25:30Z" title="Wednesday, February 5th 2020, 12:25:30 pm +01:00">Feb 5, 2020</time>_

---


Here the change is not necessary, but at the moment in the file `NotificationsTable.story.js` it can not be named `notifications`, because there this name is already defined.

We could discuss this personally. @mattwr18